### PR TITLE
More Gun Variants, Part Iv

### PR DESCRIPTION
**Pull request by DoctorBoomstick**
#### Summary
None
#### Purpose of change
Continues work left off in… #62074, over 10000 PRs ago. Wow, this was a long time coming.
We have a lot of guns. We have a variant system for non-gun-savvy people. We don’t have variants for all our guns. This is not a great state of being.
#### Describe the solution
I have added generic variants for all the firearms in .50-calibre, as well as their magazines. I have also tweeked some of the pre-existing descriptions, mainly to set some prose in the past tense to reflect the game setting, and edited the AS-50 description to make it more concise.
#### Describe alternatives you've considered
Adding variants for different guns.
#### Additional context
Consult, for prior installments of this effort, PRs:
- #62074.
- #61893.
- and #61598.

### CODE REVIEW

**Review by anothersimulacrum — COMMENTED** [review]
Can you also remove these guns from this list?
https://github.com/CleverRaven/Cataclysm-DDA/blob/f1b445998fd22a7788a67653390f93263f1acfd9/tools/json_tools/gun_variant_validator.py#L108-L111

File: data/json/items/gun/50.json [hunk 1]
```diff
@@ -31,8 +39,16 @@
     "copy-from": "rifle_auto",
     "looks_like": "modular_ar15",
```
Comment (Maleclypse) [inline data/json/items/gun/50.json:40]:
```suggestion
    "looks_like": "m107a1",
```

Comment (Maleclypse) [reply data/json/items/gun/50.json:40]: I figuered this way it at least looks more like a 50 cal weapon if there is a .50 cal rifle sprite and if not it still falls back to AR-15

File: data/json/items/gun/50.json [hunk 2]
```diff
@@ -67,8 +83,16 @@
     "copy-from": "rifle_semi",
     "looks_like": "modular_ar15",
```
Comment (Maleclypse) [inline data/json/items/gun/50.json:84]:
```suggestion
    "looks_like": "m107a1",
```

File: data/json/items/gun/50.json [hunk 3]
```diff
@@ -95,8 +119,16 @@
     "copy-from": "rifle_manual",
     "looks_like": "modular_ar15",
```
Comment (Maleclypse) [inline data/json/items/gun/50.json:120]:
```suggestion
    "looks_like": "m107a1",
```

File: data/json/items/gun/50.json [hunk 4]
```diff
@@ -123,8 +155,16 @@
...
```
Comment (Maleclypse) [inline data/json/items/gun/50.json:156]:
```suggestion
    "looks_like": "m107a1",
```

**Review by Maleclypse — APPROVED** [review]
Looks good.  I made some looks like suggestions but it's your call.  Let me know which way you decide and this is ready for merge.

**Pull request merged**